pkg/skaffold/kubernetes/portforward/entry_manager.go

```diff
@@ -89,6 +89,6 @@ func (b *EntryManager) Stop() {
 // Terminate terminates a single port forward entry
 func (b *EntryManager) Terminate(p *portForwardEntry) {
 	b.forwardedResources.Delete(p.key())
-	b.forwardedPorts.Delete(p.localPort)
+	b.forwardedPorts.Delete(int(p.localPort))
```

Comment:
Shouldn't we use `uint16` everywhere?

Reply:
So I updated to use int because a lot of the helper functions expect an int -- we've been using int32 because that's the type used for `ContainerPort` in Kubernetes, but it's only read in one place so I don't think we need to stick with it.

I can change it to uint16 if you feel strongly, but this seems to require the least amount of casting.